src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: $mainBackgroundColor;
+  width: 100%;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $mainBorderColor;
+  border-radius: 5px;
+
+  &__img {
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    color: $mainFontColor;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: $secondaryFontColor;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-reviews {
+      color: $mainFontColor;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-title {
+      color: $secondaryFontColor;
+      font-size: 12px;
+      line-height: 18px;
+      margin-top: 24px;
+    }
+
+    &-count {
+      color: $mainFontColor;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    display: flex;
+    color: $buttonTextColor;
+    background-color: $buttonBackgroundColor;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 39px;
+    text-decoration: none;
+    text-transform: uppercase;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    margin-top: 16px;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    transition: 0.2s;
```

Comment:
don't use it like that, because it's the same as transition: all 0.2s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md